diff -Naur odroid_kernel.git/arch/arm/mach-exynos/include/mach/smc.h odroid_kernel.git_new/arch/arm/mach-exynos/include/mach/smc.h
--- odroid_kernel.git/arch/arm/mach-exynos/include/mach/smc.h	2013-02-04 01:50:35.687320373 +0000
+++ odroid_kernel.git_new/arch/arm/mach-exynos/include/mach/smc.h	2013-02-04 02:10:35.267859039 +0000
@@ -51,7 +51,7 @@
 	register u32 reg3 __asm__("r3") = arg3;
 
 	__asm__ volatile (
-		".arch_extension sec\n"
+//		".arch_extension sec\n"
 		"smc	0\n"
 		: "+r"(reg0), "+r"(reg1), "+r"(reg2), "+r"(reg3)
 	);
@@ -67,7 +67,7 @@
 	register u32 reg3 __asm__("r3") = 0;
 
 	__asm__ volatile (
-		".arch_extension sec\n"
+//		".arch_extension sec\n"
 		"smc	0\n"
 		: "+r"(reg0), "+r"(reg1), "+r"(reg2), "+r"(reg3)
 	);
